lolex.js

```diff
@@ -200,9 +200,16 @@ function withGlobal(_global) {
         if (!clock.jobs) {
             return;
         }
+        // note, we can't check clock.jobs.length for > loopLimit here since jobs 
```

Comment:
Trailing space here, ci is complaining

Reply:
I was sure I fixed it - something must be wrong with the precommit hook it ran the lint and I fixed it but somehow the changes didn't make it.